include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -379,6 +379,50 @@ def TTIR_Expm1Op: TTIR_ElementwiseUnaryOp<"expm1"> {
   }];
 }
 
+class TTIR_ElementwiseUnaryWithFloatParameterOp<string mnemonic, list<Trait> traits = []> :
```

Comment:
Do we know how many ops might benefit from this common interface?

It seems like they'd all have different verifiers / interpretations of the parameter they're encoding, what do we gain? The shared builder pattern?

Reply:
Well, this is the one I wanted to get the input. Here is the list of the unary ops with the float parameter:
https://github.com/tenstorrent/tt-metal/blob/cf04a20b080b0bdd76e0b6e49f9d94fcb38350ea/ttnn/cpp/ttnn/operations/eltwise/unary/unary.hpp#L319

I also think it is too much on the dialect side to model. I am fine with the runtime part, but I need honest feedback on the dialect part.

Reply:
I see, ok there are a number of them.  I'm a bit indifferent TBH, we should make the decision based on how this will impact code maintenance and readability.  I don't see any downside to having a common builder pattern, so it's probably OK to leave it how the code reads in this PR.

Reply:
I agree, I will merge it like this then.